new-frontiers-14/frontier-station-14
# NFSD Outpost Criminal Records Fix

## About the PR
A split from https://github.com/new-frontiers-14/frontier-station-14/pull/1578


It allows proper function of criminal records by adding a workaround which lets NFSD Outpost receive a record of each player that spawned in no matter in what location, can be Frontier Outpost, Latejoining vessel etc.

This way NFSD Outpost has records of every single player, and criminal record console it is possible to write notes on player records as well as set their wanted level which works globally(from testing)

Adds system for Pirates and future antags. Where it can either fake the records, AKA real job won't be visible and replaced with one of the three most common jobs (contractor, pilot, mercenary), OR where they won't be registered in record at all.

## Why / Balance

NFSD greatly lacks managing as well as intel gathering tools. 
This will mainly assist detectives in forensics. Sherrifs in writing more accurate warrants or marking wanted/released folk.

But mostly assist with archiving infractions done by each person.


## How to test

Join any station/ship crew.
Go to NFSD Outpost
Check in station records computer if person was added
spawn criminal record console on NFSD
check if player information shows up
check if changing status has some effect

## Media


## Requirements



## Breaking changes

Anything that affects Station Records or Criminal Records.

**Changelog**


:cl:
- tweak: Criminal records computers now work sector-wide, and contain information for all employees.
